Sources/Build/BuildPlan.swift

```diff
@@ -1123,7 +1123,8 @@ public final class ProductBuildDescription {
         if buildParameters.triple.isLinux() {
             args += ["-Xlinker", "-rpath=$ORIGIN"]
         } else if buildParameters.triple.isDarwin() {
-            args += ["-Xlinker", "-rpath", "-Xlinker", "@loader_path"]
+            let rpath = product.type == .test ? "@loader_path/../../../" : "@loader_path"
```

Comment:
So this goes up out of the "MacOS" directory, then out of "Contents", then out of the test bundle, right?  i.e. it refers to the directory that contains the test bundle.

Reply:
If correct, then this looks good.

Reply:
Yep